src/util/misc/projectStroke/StrokeLineCapProjections.ts

```diff
@@ -52,7 +51,7 @@ export class StrokeLineCapProjections extends StrokeProjectionsBase {
    * @see https://github.com/fabricjs/fabric.js/pull/8344#1-1-butt
    */
   private projectButt() {
-    return [this.projectOrthogonally(this.A, this.T)];
+    return [this.projectOrthogonally(this.A, this.T, this.strokeProjectionMagnitude), this.projectOrthogonally(this.A, this.T, -this.strokeProjectionMagnitude)];
```

Comment:
I disagree.
The projection is always to the direction opposite to the vector AT

Reply:
?

Reply:
@luizzappa ?

Reply:
I got it wrong